model/task/local_service.go

```diff
@@ -121,19 +120,19 @@ func (s *localService) GetFailedTestSamples(ctx context.Context, taskOpts []Task
 func (s *localService) Get(ctx context.Context, taskOpts []Task, fields ...string) ([]testresult.TaskTestResults, error) {
 	var filter bson.M
 	if len(taskOpts) == 1 {
-		filter = createFindQuery(taskOpts[0].Id, taskOpts[0].Execution)
+		filter = byTaskIDAndExecution(taskOpts[0].Id, taskOpts[0].Execution)
 	} else {
 		findQueries := make([]bson.M, len(taskOpts))
 		for i, taskOpt := range taskOpts {
-			findQueries[i] = createFindQuery(taskOpt.Id, taskOpt.Execution)
+			findQueries[i] = byTaskIDAndExecution(taskOpt.Id, taskOpt.Execution)
 		}
 		filter = bson.M{"$or": findQueries}
 	}
 	opts := options.Find()
-	opts.SetSort(mgobson.D{
-		{Name: bsonutil.GetDottedKeyName(testresult.TestResultsInfoKey, testresult.TestResultsInfoTaskIDKey), Value: 1},
-		{Name: bsonutil.GetDottedKeyName(testresult.TestResultsInfoKey, testresult.TestResultsInfoExecutionKey), Value: 1}},
-	)
+	opts.SetSort(bson.M{
```

Comment:
This generally should be a `bson.D` to avoid the BSON map type changing the order of the keys.